eradiate/radprops/rad_profile.py

```diff
@@ -443,8 +444,9 @@ class US76ApproxRadProfile(RadProfile):
     )
 
     dataset = attr.ib(
-        default="spectra-us76_u86_4-4000_25711",
-        validator=attr.validators.in_({"spectra-us76_u86_4-4000_25711", "test"}),
+        default=None,
+        validator=attr.validators.optional(
+            attr.validators.in_(list(_AbsorptionGetter.PATHS.keys())))
```

Comment:
The default behaviour lets `compute_sigma_a` finds the absorption dataset files automatically based on the thermophysical profile and wavelength/wavenumber value. The user must leave `dataset=None` to let that happen. In the tests, we call `compute_sigma_a` with `dataset_id="test"` because we cannot rely on the absorption data being downloaded, instead we want to use a small test dataset.

I will add that to the docstrings. I agree with your remark about `compute_sigma_a` not having defaults. I'll add an issue about that.